Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:4
```

Comment:
It would be good to see if this runs on the latest version of node (`8.6.0`) when it is all running.